src/controllers/article.controller.js

```diff
@@ -368,5 +370,26 @@ export default {
       );
     }
     return Helper.successResponse(response, 201, result);
+  },
+
+  async fetchRatings(request, response) {
+    let { articleId } = request.params;
+    articleId = parseInt(articleId, 10);
+
+    const result = await fetchRatingsService(articleId);
+
+    if (result === `Article with id: ${articleId} does not exist`) {
+      return Helper.failResponse(response, 404, result);
```

Comment:
Hi @swisskid95, this check was awesome and necessary. Good work, However you didn't write the test for this check. 
Please do write a test for this edge case.
thank you.